certbot/renewal.py

```diff
@@ -301,7 +301,10 @@ def _renew_describe_results(config, renew_successes, renew_failures,
 def renew_all_lineages(config):
     """Examine each lineage; renew if due and report results"""
 
-    if config.domains != []:
+    # This is trivially False if config.domains is empty
+    if any(domain not in config.webroot_map for domain in config.domains):
```

Comment:
I think so. This is what we have been doing in the past and in general we cannot differentiate between values set on the command line and those in a CLI config file.